projects/Cura.cmake

```diff
@@ -0,0 +1,7 @@
+ExternalProject_Add(Cura
+    GIT_REPOSITORY https://github.com/ultimaker/Cura
+    GIT_TAG origin/${TAG_OR_BRANCH}
+    CMAKE_ARGS -DCMAKE_INSTALL_PREFIX=${EXTERNALPROJECT_INSTALL_PREFIX} -DCMAKE_PREFIX_PATH=${CMAKE_INSTALL_PREFIX} -DURANIUM_SCRIPTS_DIR= -DCURA_VERSION=${CURA_VERSION}
```

Comment:
The standard build should not generate any .mo files but just use those from cura-binary-data. Specifying an empty scripts dir is currently the way to tell that to Cura's build system.